tests/testdata/tools/k8s-spot-termination-handler/main.tf

```diff
@@ -1,17 +0,0 @@
-provider "kubernetes" {
```

Comment:
Deleted this since it was removed in https://github.com/liatrio/lead-terraform/pull/420